lib/pbench/client/__init__.py

```diff
@@ -302,8 +304,13 @@ def delete(
         return response
 
     def connect(self, headers: Optional[dict[str, str]] = None) -> None:
-        """Connect to the Pbench Server host using the endpoints API to be sure
-        that it responds, and cache the endpoints response payload.
+        """Performs some pre-requisite actions to make server client usable.
+
+            1. Connect to the Pbench Server host using the endpoints API to be
+            sure that it responds, and cache the endpoints response payload.
+
+            2. Creates an OIDCAdmin object that server client can use to
+            perform admin privileged actions on OIDC server.
```

Comment:
- It would be nice if `Connect` (and `cache`) and `Creates` used a consistent tense/voice/mood.  (E.g., s/Creates/Create/ )
- s/that server client can/that a server client can/, or
s/that server client can/that server clients can/
- I would keep only one of `admin` or `privileged` (unless you want to hyphenate them).
- s/on OIDC server/on an OIDC server/